source/includes/seq-edit-document.rst

```diff
@@ -0,0 +1,9 @@
+1. At the top of this document, click :guilabel:`Edit Document`.
+   |vsce| opens it as an editable |ejson| document titled ``<database>.<collection>:"<_id>".json``.
+#. Make any edits you require.
+#. Press :kbd:`Ctrl + S` (Windows/Linux) or :kbd:`Cmd + S` to save the
+   edited document to the MongoDB database.
+
+   - If the update results in an error, |vsce| displays it.
```

Comment:
Can we reverse these unordered bullets? Success first, then error?

Reply:
Agreed. Fixed.